### DISCUSSION TITLE
"The current platform is unsupported" when installing lua-language-server on aarch64 alpine linux docker

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [mason-org/mason.nvim#1406] [Q&A] [answered]
I am using `OrbStack` on macOS, and I created a docker container which is from `alpine`. The result of `uname -a` is:

```
Linux 96c7f1abfc9f 6.3.12-orbstack-00209-ga2cd8129a099 #1 SMP Mon Jul 10 05:35:25 UTC 2023 aarch64 Linux
```

Inside that container, I installed a nvim with mason.nvim. However, it failed to install `lua-language-server` and raised the error:

> The current platform is unsupported

I noticed the [release page](https://github.com/LuaLS/lua-language-server/releases) shows that there is an arm64 linux package.

**Comment by williamboman** [comment, marked as answer]
Those binaries are linked to glibc which is not readily available on Alpine. If you have it installed you can bypass this by `:MasonInstall --target linux_arm64_gnu lua-language-server`

**Reply by ChenZhongPu** [reply]
`stylua` cannot be installed due to the similar reason, but this package is found at [Alpine official repo](https://pkgs.alpinelinux.org/packages?name=stylua&branch=v3.18&repo=&arch=&maintainer=). Is there any way to specify the download source using Mason?

**Reply by williamboman** [reply]
No you can't change to arbitrary download sources

**Comment by AidanWelch** [comment]
@williamboman 's answer is correct but seems to be slightly outdated, I had to do `:MasonInstall --target=linux_arm64_gnu lua-language-server`
